src/rest/StreamEndpoints.ts

```diff
@@ -248,7 +273,7 @@ export class StreamEndpoints {
         })
 
         // Send data to the stream
-        return authFetch(
+        await authFetch(
```

Comment:
why not return?

Reply:
Because the success response `POST /​streams/​id/​data` is always empty. So it is better to return void than an empty object. Error response rejects the promise.